src/altinn-app-frontend/src/types/index.ts

```diff
@@ -276,6 +307,15 @@ export interface IFetchSpecificOptionSaga {
   instanceId?: string;
 }
 
+export interface IFetchSpecificDataListSaga {
+  dataListId: string;
+  formData?: IFormData;
+  language?: string;
+  dataMapping?: IMapping;
+  secure?: boolean;
+  instanceId?: string;
+}
```

Comment:
I think most of the types defined in this file should probably live closer to their implementations, right? Try putting them somewhere like `shared/resources/dataLists/index.d.ts` instead of here.